### PR TITLE
Remove -e from repo requirements

### PR DESCRIPTION
**Pull request by nickpell**
https://github.com/jazzband/pip-tools/issues/355 is resolved by https://github.com/jazzband/pip-tools/pull/807 so we don't need the ```-e``` anymore.

### CODE REVIEW

File: requirements/requirements.in [hunk 1]
```diff
@@ -4,7 +4,7 @@ jsonobject==0.9.9
 jsonobject-couchdbkit~=0.9
 # celery 4.1.1 forked to work around https://github.com/celery/celery/issues/4737
 # upgrade when https://github.com/celery/celery/issues/4980 is fixed
--e git+git://github.com/nickpell/celery@dc7253614fd8500af78b09d1ceb79c50f1baac8d#egg=celery
+git+git://github.com/nickpell/celery@dc7253614fd8500af78b09d1ceb79c50f1baac8d#egg=celery
```
Comment (emord) [inline requirements/requirements.in]: should this have a version that goes with it? `egg=celery==4.1.1b1` or something like that?

Comment (nickpell) [reply requirements/requirements.in]: Should it?  I know we hypothesized this earlier but it seems to be working fine without it.

Comment (emord) [reply requirements/requirements.in]: so long as the previous releases continue to work it seems fine. TBH I'm not super sure why I included that with architect

**Review by emord — APPROVED** [review]

**Review by calellowitz — APPROVED** [review]
do we need to update pip-tools somewhere to get this to work?

**Review by dannyroberts — APPROVED** [review]
Nice!

**Comment by nickpell** [comment]
@calellowitz pip-tools is already upgraded: https://github.com/dimagi/commcare-hq/pull/24514

**Pull request merged**